sandbox/interactivity-sandbox.js

```diff
@@ -161,10 +166,20 @@ function sandbox() {
 
 function save_current_code(code) {
     if (!localStorage) throw new Error('Local storage support required');
-    localStorage.setItem('_current_code', code);
+    localStorage.setItem('_i_current_code', code);
 }
 
 function load_last_code() {
     if (!localStorage) throw new Error('Local storage support required');
-    return localStorage.getItem('_current_code') || '';
+    return localStorage.getItem('_i_current_code') || '';
+}
+
+function save_current_json(json) {
+    if (!localStorage) throw new Error('Local storage support required');
```

Comment:
http://caniuse.com/#search=localstorage
don't think that is necessary. also, in the event `localStorage` is not defined, you'll get a `ReferenceError` for an undefined element here, so `window.localStorage` will have to be used.